src/cmd/config.go

```diff
@@ -15,6 +15,7 @@ var setupFlag bool
 var configCommand = &cobra.Command{
 	Use:   "config",
 	Short: "Displays or updates your Git Town configuration",
+	Long:  `Displays or updates your Git Town configuration`,
```

Comment:
Agreed. I have removed it. Notice I changed the documentation to `Displays or resets your Git Town configuration`